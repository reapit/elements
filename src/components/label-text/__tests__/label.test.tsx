import { render } from '@testing-library/react'
import { LabelText } from '..'

describe('Label component', () => {
  it('should match a snapshot', () => {
    const wrapper = render(<LabelText />)
    expect(wrapper.asFragment()).toMatchSnapshot()
  })

  it('should match a snapshot if it required', () => {
    const wrapper = render(<LabelText isRequired />)
    expect(wrapper.asFragment()).toMatchSnapshot()
  })
})
