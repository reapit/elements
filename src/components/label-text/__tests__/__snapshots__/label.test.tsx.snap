// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Label component > should match a snapshot 1`] = `
<DocumentFragment>
  <span
    class="mocked-styled-0 el-label-text"
    data-size="medium"
    data-variant="soft"
  />
</DocumentFragment>
`;

exports[`Label component > should match a snapshot if it required 1`] = `
<DocumentFragment>
  <span
    class="mocked-styled-0 el-label-text"
    data-size="medium"
    data-variant="soft"
  >
    <span
      class="mocked-styled-1 el-label-required-mark"
    >
      *
    </span>
  </span>
</DocumentFragment>
`;
